### ISSUE TITLE
Tracker Doesn't Update even after "Full Reset"

### ISSUE THREAD

**Issue by lamprador** [Razviar/mtga-pro-tracker#420]
Hi, my profile page is: https://mtgarena.pro/u/Agha
It seems the tracker doesn't update stats since the May 24 MTGA update.
I'm running on W10 64.
I tried "Full Reset", "Manual Re-Sync", "Scan Old Logs", manually "Locating Game Log" and performing the previous steps, reinstalling MTGArena.PRO tracker, reinstalling MTGArena, runing the tracker as administrator and lastly granting full permissions and/or changing ownership on the folders running the tracker and/or the log file location.
The steps of the things i tried are more or less as the order i write them above.
Edit1: Something more i should add is that when i try to sync or scan logs in any way is that it says "Reading Log" and after a few seconds it goes back to "Awaiting for Updates" but it doesn't say uploading data with the update bar updating as it used to.
Edit2: My .net Framework version is 4.8 and MTGApro has both Private and Public access through Firewall.
I updated the tracker version today, tried resyncing and it still doesn't update my data.
Any suggestions on what else i can do?
Thank you in advance.

**Comment by lamprador** [comment]
Something more i should add is that when i try to sync or scan logs in any way is that it says "Reading Log" and after a few seconds it goes back to "Awaiting for Updates" but it doesn't say uploading data with the update bar updating as it used to.

**Comment by Lumeni77** [comment]
Tracker is not updating for me either since 1.6.9. When doing manual resync it sometimes does some of the updates, like collections. But boosters are not updated and a lot more.

**Comment by Aquaman88** [comment]
Thank you for your messages. Did you upload last updates? Can you add screenshots of your problem here?

**Comment by lamprador** [comment]
I managed to upload yesterday and get the tracker to work. It seems that the problem is on my end, because i run it on the windows hidden administrator user and everything worked properly, but i can't seem to run it on my user although i am an administrator and have access everywhere. Weird but it has something to do with Windows and not you, so i have to figure it out somewhere else i guess.
Thank you for your reply and sorry to bother you.

**Comment by Lumeni77** [comment]
I closed it a few times and restarted the app. At first it still didn't work. At some point I decided not to minimize and let it be open in mij 2nd screen. In the beginning it dit not recognize updates while playing. Even forcing synd did nothing. And then after another restart and having it on my 2nd screen it start updating again. Later I tried mnimizing and it worked. After this I did not have the same problem again. Don't know what started it and what solved it.
